apidoc/Titanium/UI/RefreshControl.yml

```diff
@@ -37,7 +41,14 @@ properties:
 
 events:
   - name: refreshstart
-    summary: Fired in response to a user initiated action to refresh the contents of the table view or list view.
+    summary: Fired in response to a user initiated an action to refresh the contents of the table view, list view or scroll view.
+    properties:
+      - name: bubbles
+        summary: This is false. This event does not bubble
+        type: Boolean
+  - name: refreshend
+    summary: Fired in response to a user finished an action to refresh the contents of the table view, list view or scroll view.
```

Comment:
It is the native description Apple provides and is copied & adjusted from `refreshstart`.